docs/lint_errors.md

```diff
@@ -220,6 +221,24 @@ This test will fail if the following requirements are not met in these files:
             { [[ ${{github.event.pull_request.head.repo.full_name}} == <repo_name>/<pipeline_name> ]] && [[ $GITHUB_HEAD_REF = "dev" ]]; } || [[ $GITHUB_HEAD_REF == "patch" ]]
       ```
 
+4. `awstest.yml`: Triggers tests on AWS batch. As running tests on AWS incurs costs, they should be only triggered on `push` to `master` and `release`.
+    * Must be turned on for `push` to `master` and `release`.
+    * Must not be turned on for `pull_request` or other events.
+
+### GitHub Actions AWS full tests
+
+Additionally, we provide the possibility of testing the pipeline on full size datasets on AWS.
```

Comment:
Maybe have these two sentences on a single line (maybe this is just Github being mean), then having the list of things the tests contain?  (Similar to above, found this easier to read)

Reply:
This is one thing that annoys me on GitHub. I prefer writing markdown with one sentence per line, as diffs are much easier to read for minor changes. But GitHub-flavoured markdown renders a single linebreak with a `<br>`, which is different to basically all other markdown engines.

I guess we expect the majority of people to be reading this on the nf-core website (linked to from the lint tests), so I'm not sure that it matters much if it's a little tricky to read on github.com

Reply:
yes, exactly, I was writing each sentence in a new line to facilitate the git diffs and it should render as a paragraph on the nf-core website :)

Reply:
Cool with me then, I always get annoyed because of that behaviour and just wanted to make sure this is fine which it seemingly is :-) Docs are good to me :-)